Tasks/UniversalPackagesV0/universalpublish.ts

```diff
@@ -49,16 +50,9 @@ export async function run(artifactToolPath: string): Promise<void> {
 
         if (feedType === "internal")
         {
-            // getting inputs
-            serviceUri = tl.getEndpointUrl("SYSTEMVSSCONNECTION", false);
 
-            packageName = tl.getInput("packageListPublish");
-            const feedProject = getProjectAndFeedIdFromInputParam("feedListPublish");
-            feedId = feedProject.feedId;
-            projectId = feedProject.projectId;
+            [serviceUri, packageName, feedId, projectId, accessToken] = authSetup(feedType);
```

Comment:
since this is the same in both legs of the `if`, consider hoisting it out to the outer scope, above the `if`